translations/fr/try_ruby_170.md

```diff
@@ -0,0 +1,37 @@
+---
+lang:   FR
+title:  Élementaire
+answer: ^(12|35|47)$
+load:   ticket = [12, 47, 35]
+ok:     D'accord
+error:  Presque
+---
+
+Donc nous avons stocké nos numéros de loterie dans la variable ticket.
+Comment faire pour les retrouver ?
+
+On a déjà vu qu'il est possible d'obtenir le numéro le plus élevé avec __max__.
+On peut également obtenir le premier (__first__) ou le dernier (__last__)
+élement de la liste.
+Mais que faire si on a besoin d'un élement specifique de la liste?
+
+### [ ]
+Ruby utilise des crochets droits [ ] pour viser un élement.
```

Comment:
In french I think we would simply say `crochets` for `[]` (and `accolades` for `{}`).